src/SdpGenerator.c

```diff
@@ -269,7 +269,11 @@ static PSDP_OPTION getAttributesList(char*urlSafeAddr) {
     // Send client feature flags to Sunshine hosts
     if (IS_SUNSHINE()) {
         uint32_t moonlightFeatureFlags = ML_FF_FEC_STATUS | ML_FF_SESSION_ID_V1;
+#ifdef __3DS__
+        snprintf(payloadStr, sizeof(payloadStr), "%lu", moonlightFeatureFlags);
```

Comment:
Is `uint32_t` not `unsigned int` on 3DS?

Reply:
I pulled this change from https://github.com/moonlight-stream/moonlight-common-c/pull/84, and the compiler warnings seemed to agree with the change. When I print the `sizeof(unsigned int)` though, it actually reports that its 4 bytes so... maybe this is some weird compiler thing? Now that I've checked the variable size in the actual program, I feel safe reverting this change.